package.json

```diff
@@ -15,11 +15,12 @@
   "license": "MIT",
   "private": false,
   "scripts": {
-    "prettier-format": "prettier --loglevel 'warn' --write './src/**/*.js'",
-    "prettier-diff": "prettier --loglevel 'warn' --check './src/**/*.js'",
+    "prettier-format": "prettier --loglevel 'warn' --write './src/**/*.ts'",
+    "prettier-diff": "prettier --loglevel 'warn' --check './src/**/*.ts'",
     "build": "swc ./src -d dist",
     "build-watch": "swc ./src -d dist -w",
-    "test": "jest ./src"
+    "test": "jest ./src",
+    "type-check": "tsc --noEmit"
```

Comment:
Prepared a script for type-checking because of this reason: https://github.com/Fullscript/eslint-plugin/pull/14#discussion_r1554362665